dev_guide/application_lifecycle.adoc

```diff
@@ -0,0 +1,88 @@
+= Application Life Cycle Examples
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As a PaaS, OpenShift is designed for building and deploying applications.
+Depending on how much you want to involve OpenShift in the development process,
+you can choose to develop on OpenShift and use it to continuously develop an
+application, or you can deploy a fully developed application onto a OpenShift
+instance.
+
+== Developing on OpenShift
+
+You can develop your application on OpenShift directly. Use the following if you
```

Comment:
I'd prefer "Use the following _____ if ...." where ____ is something like "wizard" or "guidelines" or "decision-making process" or "questions" or ...